docs/cookbooks/BasicRAG.md

```
@@ -121,5 +128,8 @@ async with ctx.scope(
         tools=Toolbox.of(index_search_tool, suggest=index_search_tool),
     )
     print(result)
+````
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)